phys/module_cu_ntiedtke.F

```diff
@@ -2223,12 +2253,18 @@ subroutine cuascn &
       real     zrnew,zz,zdmfeu,zdmfdu,dp
       real     zfac,zbuoc,zdkbuo,zdken,zvv,zarg,zchange,zxe,zxs,zdshrd
       real     atop1,atop2,abot
+      real     scale_fac(klon)
+      integer  ntiedtke_dx_opt
 !--------------------------------
 !*    1.       specify parameters
 !--------------------------------
       zcons2=3./(g*ztmst)
       zfacbuo = 0.5/(1.+0.5)
+      if(ntiedtke_dx_opt.eq.1) then
+        zprcdgw = cprcon*zrg/(scale_fac(jl)**(1./3.))
+      else
```

Comment:
Why is scale_fac defined as the cube of something and then used as the cube root?

Reply:
I see now it is used in other places.

Reply:
also in the paper I think it says square root?

Reply:
@dudhia Thanks for pointing it out. That is an error in the paper. It is the cubic root that has been used in the code.